src/Microsoft.ServiceFabric.AspNetCore/Microsoft.ServiceFabric.AspNetCore.csproj

```diff
@@ -18,7 +18,11 @@
     <!-- Direct dependencies -->
     <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="2.2.7" />
```

Comment:
Are we on the latest versions for other packages?

Reply:
Yes. It is no longer maintained, and this is the lates version available. ASP.NET Core libraries are now shipped as part of .NET SDK.

Reply:
I believe the 2.2 versions are not supported anymore and it is visible on nuget.org - 
![image](https://github.com/user-attachments/assets/53269b81-7894-4f2b-abc1-c1c54000508b)

While for 2.1.1 is doesn't show a warning because it is still supported as part of [ASP.NET Core on .NET Framework](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core#:~:text=November%2012%2C%202024.-,ASP.NET%20Core%202.1%20on%20.NET%20Framework,-Support%20for%20ASP)
![image](https://github.com/user-attachments/assets/b9830caa-6463-472e-ae3e-24036c74a304)

Reply:
@IvanAlekseev thank you for pointing this out. These changes will be made in a separate PR, as we need to do some investigation prior to incorporating them.

Reply:
Somewhat related: [ASP.NET Core 2.1 becomes ASP.NET Core 2.3](https://github.com/dotnet/announcements/issues/331)